content/docs/deployment/csmoperator/drivers/unity.md

```diff
@@ -0,0 +1,92 @@
+
+2. Create a file called `config.yaml` that has Unity XT array connection details with the following content
+   ```yaml
+      storageArrayList:
+      - arrayId: "APM00******1"                 # unique array id of the Unisphere array
+        username: "user"                        # username for connecting to API
+        password: "password"                    # password for connecting to API
+        endpoint: "https://10.1.1.1/"           # full URL path to the Unity XT API
+        skipCertificateValidation: true         # indicates if client side validation of (management)server's certificate can be skipped
+        isDefault: true                         # treat current array as a default (would be used by storage classes without arrayID parameter)
+   ```
+   Change the parameters with relevant values for your Unity XT array. 
+   Add more blocks similar to above for each Unity XT array if necessary.
+           
+3. Use the following command to create a new secret unity-creds from `secret.yaml` file.
+	
+    `kubectl create secret generic unity-creds -n unity --from-file=config=secret.yaml`
+    
+   Use the following command to replace or update the secret:
+    
+    `kubectl create secret generic unity-creds -n unity --from-file=config=secret.yaml -o yaml --dry-run | kubectl replace -f -`
+
+### Install Driver
+
+1. Follow all the [prerequisites](#prerequisite) above
+   
+2. Create a CR (Custom Resource) for Unity XT using the sample files provided 
+   [here](https://github.com/dell/csm-operator/tree/master/samples). This file can be modified to use custom parameters if needed.
+
+3. Users should configure the parameters in CR. The following table lists the primary configurable parameters of the Unity XT driver and their default values:
+
+| Parameter | Description | Required | Default |
+| --------- | ----------- | -------- |-------- |
+| replicas | Controls the number of controller pods you deploy. If the number of controller pods is greater than the number of available nodes, the excess pods will be in pending state until new nodes are available for scheduling. Default is 2 which allows for Controller high availability. | Yes | 2 |
+| namespace | Specifies namespace where the driver will be installed | Yes | "unity" |
+| fsGroupPolicy | Defines which FS Group policy mode to be used. Supported modes `None, File and ReadWriteOnceWithFSType` | No |"ReadWriteOnceWithFSType"|
+| ***Common parameters for node and controller*** |
+| X_CSI_UNITY_NODE_NAME_PREFIX | Prefix to add to each node registered by the CSI driver | Yes | "csi-node" 
+| X_CSI_UNITY_ALLOW_MULTI_POD_ACCESS | To enable sharing of volumes across multiple pods within the same node in RWO access mode | No | false |
+| X_CSI_UNITY_SYNC_NODEINFO_INTERVAL | Time interval to add node info to array. Default 15 minutes. Minimum value should be 1 | No | 15 |
+| CSI_LOG_LEVEL | Sets the logging level of the driver | true | info |
```

Comment:
Default value is debug

Reply:
For unity the default value is info as per the driver documentation

Reply:
But in csm-operator code, i see it set to debug for all drivers